statistics/handle/cache/statscacheinner.go

```diff
@@ -129,9 +164,15 @@ func (sc *StatsCache) Cost() int64 {
 
 // SetCapacity sets the memory capacity of the cache.
 func (sc *StatsCache) SetCapacity(c int64) {
+	metrics.CostGauge.Set(float64(c))
```

Comment:
We updated this both here and in `(*LFU).SetCapacity()`. I think this is unintended.

Reply:
If we read the code from the coprocessor cache, we will know the capacity is from the cost and the user can config the max memory usage of the coprocessor. this LFU is the same approach.

Reply:
the function can be changed as ```SetCost```.